content/docs/configuration/acme/README.md

```diff
@@ -383,4 +343,28 @@ solvers:
 ```
 In this case the `DNS01` solver for CloudFlare will only be used to solve a
 challenge for a DNS name if the `Certificate` has a label from
-`matchLabels` _and_ the DNS name matches a zone from `dnsZones`.
\ No newline at end of file
+`matchLabels` _and_ the DNS name matches a zone from `dnsZones`.
+
+
+## Alternative Certificate Chains
+
+{/* This empty link preserves old links to #alternative-certificate-chain", which matched the old title of this section */}
+
+<a id="alternative-certificate-chain" className="hidden-link"></a>
+
+It's possible to choose alternative certificate chains when fetching a certificate from an ACME server. This allows issuers to gracefully roll people over to a new root.
+
+This functionality is not Let's Encrypt exclusive; if your ACME server supports signing by multiple CAs you can use `preferredChain` with the value of the Common Name of the chain you want in the Issuer part of the certificate.
+
+By way of an example, below is how a user would have requested an alternative chain before the [now completed](https://community.letsencrypt.org/t/transition-to-isrgs-root-delayed-until-jan-11-2021/125516) "ISRG Root" changeover. Since this change has already happened, there's no need for this with Let's Encrypt any more:
```

Comment:
```suggestion
By way of an example, below is how a user would have requested an alternative chain before the (now completed) ["ISRG Root" changeover](https://community.letsencrypt.org/t/transition-to-isrgs-root-delayed-until-jan-11-2021/125516). Since this change has already happened, there's no need for this with Let's Encrypt anymore:
```

Reply:
I prefer the link that way yeah, thanks :+1: